### PR TITLE
fix(test): Fix presubmit with python version upgrade

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**
E2E test, sample test and upgrade test are failing because of python 3.5 not supporting f"string" context. This PR is for fixing the test failure issue which might have been blocking everyone.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by capri-xiyue** [comment]
@Bobgy  Can you help approve it?

File: backend/Dockerfile [hunk 1]
```diff
@@ -32,10 +32,10 @@ COPY backend/src/apiserver/config/sample_config.json /samples/
 # The default image is replaced with the GCR-hosted python image.
 RUN set -e; \
     < /samples/sample_config.json jq .[].file --raw-output | while read pipeline_yaml; do \
-        pipeline_py="${pipeline_yaml%.yaml}"; \
```
Comment (capri-xiyue) [inline backend/Dockerfile:35]: Why do we change the format here?

Comment (zijianjoy) [reply backend/Dockerfile:35]: This is auto-formated by VS code when I am editing this file. It seems to be accurate formatting, should I revert this format change?

Comment (Bobgy) [reply backend/Dockerfile]: I've edited the branch directly, the extra indent was intentional to make the while loop easier to read. So it seems reasonable to reserve it.

**Comment by chensun** [comment]
/lgtm
/approve

Thanks, James.

File: backend/Dockerfile (hunk 1 above)
Comment (zijianjoy) [reply backend/Dockerfile]: Sounds good, I will reserve the original formatting here.

**Pull request merged**